### ISSUE TITLE
A.vec 

### ISSUE THREAD

**Issue by chrislouis0106** [xwhan/DeepPath#22]
hi,
can you share the work of A.vec and how to get the A.vec file

**Comment by teenkevo** [comment]
> hi,
> can you share the work of A.vec and how to get the A.vec file

This code is really hard to follow, I dont think the authors will ever provide any detailed documentations for it

**Comment by chrislouis0106** [comment]
ENN，no matter what items for this code, it is terrible and half-open source. thank you



---Original---
